gui/src/app/StanSampler/StanSampler.ts

```diff
@@ -27,7 +27,10 @@ class StanSampler {
     #onProgressCallbacks: ((progress: Progress) => void)[] = [];
     #onStatusChangedCallbacks: (() => void)[] = [];
     #draws: number[][] = [];
+    #computeTimeSec: number | undefined = undefined;
     #paramNames: string[] = [];
+    #numChains: number = 0;
```

Comment:
It's useful to attach it to the sampler so that we know for sure what was the configuration when the sampler was run. It is used in the `useSamplerOutput` hook that takes only the `sampler` as input argument.

Reply:
Good point, I didn’t think about what would happen if someone changed the sampler settings but hadn’t clicked the re-run button yet